### DISCUSSION TITLE
DB Entities

### DISCUSSION THREAD

**Discussion by Just-gomin** [imimtu/Brick-Back#24] [Ideas]
@Seungyeup 
- [현재 ERD](https://github.com/imimtu/Brick-Back/blob/main/document/Brick_ER_Diagram.drawio.svg)

여기 Brick에 created_by, modified_by가 있고,
UserAccount에도 created_by랑 modified_by 칼럼들이 있는데, UserAccount에는 해당 칼럼들이 어떤 이유에서 필요한가 싶어서 질문남겨.
어떤 것 때문에 헤딩 칼럼이 생성된거야??

그리고, hash Tag 따로 테이블 빼는거에 관한것도 토의가 필요할 것 같은데 언제 가능할까??

**Comment by Seungyeup** [comment]
@Just-gomin 

운영상의 이슈(예를들어 가입 1주년 고객 축하포인트 지급 등 가입 시점과 관련된 이벤트 진행 등, 해당 유저의 정보가 바뀐 경우 관리자가 개입했는지 확인 ),
성능상의 이슈(db 조회 등을 할 때, 해당 유저의 생성시점 이후를 검색조건 으로 넣으면 범위가 팍 줄겠지? 인덱스를 태워도)
등이 있는데 사실 그렇게 깊게 고민했던 포인트는 아니고 비용이 크게 차이나지 않는 선에서 모든 테이블에 공통으로 로깅할 생각이야!

+ hashtag의 경우 별도 테이블로 가져가려면 커스텀 가능하게 할건지?(지난번 얘기나온것처럼)등등 의견 여기 바로 남겨주시죠! 엔티티 예시로 정의해서 남겨도 좋고

**Comment by Just-gomin** [comment]
@Seungyeup 
- 생성자 및 수정자는 운영상의 이슈를 가정하고 만들었다고 이해하면 되는군..! 고마워ㅋㅋ
- hashtag 부터 해서 그러면 우리 Documents Repository 따로 파서 진행할께!

**Comment by Seungyeup** [comment]
@Just-gomin  내일 만나기 전에 ㅋㅋㅋㅋㅋ

- 우리 지금 현재 바라보고 있는 기본 ERD가 [이거](https://app.diagrams.net/#Himimtu%2FDocuments%2Fmain%2FBrick%2FERD%2FBrick_ER_Diagram.drawio) 로 생각하면 되지?? 특히 HashTag 별도 테이블로 분리하는 부분
짬날때 조금씩 건드리다보니 기억이 안나서리 👍🏼

**Comment by Just-gomin** [comment]
응 ㅋㅋ맞아 저거 기준으로!
